share/completions/climate.fish

```diff
@@ -0,0 +1,81 @@
+# fish completion for climate
+# climate is a tool that provide simple commands that wrap some useful
+# but complex combination of Linux commands.
+# The tool can be found at https://github.com/adtac/climate
+
+function __fish_is_git_repository
```

Comment:
That one _prints_ the working directory, though completion conditions are implicitly silenced.

Yeah, we should extract it. I can do that later though.